esphome/components/seeed_mr24hpc1/number/__init__.py

```diff
@@ -0,0 +1,132 @@
+import esphome.codegen as cg
+from esphome.components import number
+import esphome.config_validation as cv
+from esphome.const import (
+    ENTITY_CATEGORY_CONFIG,
+)
+from .. import CONF_MR24HPC1_ID, MR24HPC1Component, mr24hpc1_ns
+
+SensitivityNumber = mr24hpc1_ns.class_("SensitivityNumber", number.Number)
+CustomModeNumber = mr24hpc1_ns.class_("CustomModeNumber", number.Number)
+ExistenceThresholdNumber = mr24hpc1_ns.class_("ExistenceThresholdNumber", number.Number)
+MotionThresholdNumber = mr24hpc1_ns.class_("MotionThresholdNumber", number.Number)
+MotionTriggerTimeNumber = mr24hpc1_ns.class_("MotionTriggerTimeNumber", number.Number)
+Motion2RestTimeNumber = mr24hpc1_ns.class_("Motion2RestTimeNumber", number.Number)
```

Comment:
This should be named "properly"
```suggestion
MotionToRestTimeNumber = mr24hpc1_ns.class_("MotionToRestTimeNumber", number.Number)
```